start: Add a parameters hidden section to experiments to merge params/metrics into experiments (#3163)
iterative/dvc.org: content/docs/start/experiments.md

<details>

### ℹ️ More information about (Hyper)parameters

It's pretty common for data science projects to include configuration files that
define adjustable parameters to train a model, adjust model architecture, do
pre-processing, etc. DVC provides a mechanism for stages to depend on the

> specific variables from a file.

Reviewer: > DVC provides a mechanism for stages to...

What are stages? First time that word is mentioned in this page. Let's generalize 🙂

Author: stages -> experiments in c0c233065

> By default, DVC assumes a parameters file named `params.yaml` is available in

Reviewer: CE
```suggestion
By default, DVC assumes that a params file named `params.yaml` is available in
```

Author: modified in 984e03b5a

your project. DVC parses this file and creates dependencies to the variables

> (`model.conv_units` and `train.epochs`) found in the file.

Reviewer: ce
```suggestion
found in it: `model.conv_units` and `train.epochs`.
```

Author: modified in ea8c4ec89

> Here is the contents of `params.yaml` file:

Reviewer: 💅 could skip this sentence/paragraph by just adding "Example:" in the previous paragraph.

Author: Done so in 800a11cfe

```yaml
train:
  epochs: 10
model:
  conv_units: 16
```

> When you use `--set-param` option for `dvc exp run`, DVC updates these values

Reviewer 2: `updates these values` might sound like we are updating the params file itself?

Author: We are updating the values in params file. Will add "values" there.

> When you use `--set-param` option for `dvc exp run`, DVC updates these values

Reviewer: ce
```suggestion
When you use `dvc exp run --set-param`, DVC updates these values
```
I think after https://github.com/iterative/dvc.org/pull/3140 this will link to the right place 🙂

Author: Revised in 4c80077
